tests/unit/components/pacbio/PacbioLibraryCreate.spec.js

```diff
@@ -99,50 +93,52 @@ describe('PacbioLibraryCreate.vue', () => {
       expect(wrapper.element.querySelector('#pacbioLibraryCreate')).toBeNull()
     })
 
-    it('should call the createLibrary method when the create button is clicked', async () => {
-      modal.createLibrary = vi.fn()
+    it('should call the create method when the create button is clicked', async () => {
+      cmp.createLibrary = vi.fn()
       wrapper.find('#pacbioLibraryCreate').trigger('click')
       await nextTick()
       wrapper.find('#create-btn').trigger('click')
-      expect(modal.createLibrary).toBeCalled()
+      expect(cmp.createLibrary).toBeCalled()
     })
 
     it('should not display library form when the cancel button is clicked', async () => {
       wrapper.find('#pacbioLibraryCreate').trigger('click')
       await nextTick()
-      expect(modal.isDisplayLibraryForm).toBeTruthy()
+      expect(cmp.isDisplayLibraryForm).toBeTruthy()
       wrapper.find('#cancel-btn').trigger('click')
       await nextTick()
```

Comment:
this test is a bit weird, it checks the displayform variable is true as a means of checking the library form is displayed, then checks the library form element does not exist as a means of checking the library is not displayed. I think both checks should either be for the '#libraryForm' element or for the modal variable (or both).

Reply:
It is weird. I almost feel like removing all 3 of these tests? as they are testing implementation and probably sufficiently tested elsewhere and in the e2e tests.

Reply:
Happy to have them removed if they are covered in e2e 👍